src/components/organisms/sidebar/userMenu.tsx

```diff
@@ -31,6 +36,23 @@ export const UserMenu = ({ openFeedbackForm }: { openFeedbackForm: () => void })
 		close();
 	};
 
+	const getStatusIndicatorClasses = (status?: MemberStatus) =>
```

Comment:
Add more padding from the right:
![image](https://github.com/user-attachments/assets/229a166e-ba35-425a-9c9b-4558e5174821)